xep-0321.xml

```diff
@@ -55,7 +55,7 @@
   <p>This document addresses the following requirements:</p>
   <ol>
     <li>Make it possible for remote services or entities to manage user's roster by the same mechanisms that descibed in the &rfc6121;.</li>
-    <li>Provide a way for user to control which services have permission to manage his roster.</li>
+    <li>Provide a way for user to control which services have permission to manage their roster.</li>
```

Comment:
+1 for `Provides` and `Makes`.

I'm not referring to the singular, non-gendered usage of `their`, I'm referring to the way user is referenced without an article, which IMO is not proper English (unless `user` is used as a proper noun, but then it should be capitalized).

So `Provide a way for user` should be `Provides a way for the user` or `Provides a  way for the user` (either is fine).

Similarly there's a missing article in the previous sentence.